packages/react-components/src/templates/EventPage.tsx

```diff
@@ -78,9 +100,15 @@ const EventPage: React.FC<EventPageProps> = ({
       {eventConversation}
       {relatedResearch && relatedResearch?.length > 0 && (
         <RelatedResearchCard
-          description="Find out all shared research outputs that are related to this event."
+          title={titleOutputs}
```

Comment:
I believe this requirement from https://asaphub.atlassian.net/browse/GP2HUB-112

> Display 3 rows of events, if there are more than 3 rows of related outputs then display show more/show less functionality

is missing